target/linux/ath79/dts/qca9531_comfast_cf-e5.dts

```diff
@@ -0,0 +1,170 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
+
+/ {
+	compatible = "comfast,cf-e5", "qca,qca9531";
+	model = "COMFAST CF-E5/E7";
+
+	keys {
+		compatible = "gpio-keys-polled";
+
+		poll-interval = <20>;
+		pinctrl-names = "default";
+		pinctrl-0 = <&jtag_disable_pins &led_wan_pin>;
```

Comment:
@fall513 You're not using GPIOs 0-3 in `keys` node, but you're using them for LEDs, so please move the pinctrl under `leds` node.